app/scripts/views/tooltip.js

```diff
@@ -27,6 +27,14 @@ define([
       this.dismissible  = options.dismissible || false;
       this.extraClassNames = options.extraClassNames || '';
 
+      // Event codes for the keyboard keys.
+      this.DOWN_ARROW_KEY_CODE = 40;
```

Comment:
In the future, we may want to create a lib/key-codes.js to add these codes and they can be shared. We have `13` hard coded in a bunch of files IIRC.